munit/js/src/test/scala-2.11/hedgehog/munit/HedgehogAssertionsSuite.scala

```diff
@@ -0,0 +1 @@
+/home/jackcviers/Documents/development/scala-hedgehog/munit/js/src/test/scala-3.0/hedgehog/munit/HedgehogAssertionsSuite.scala
```

Comment:
It's a link to your local machine.
When I run `ls -lR munit/*/target/scala-*/*classes/hedgehog/munit` at the project root, I don't see class files generated for all these symlinks.

Reply:
Yep. I did it with emacs, which used the local paths. I'll fix.

Reply:
Addressed in https://github.com/hedgehogqa/scala-hedgehog/pull/229/commits/f3b92e35c18d0c3b77846970d1668d8a8c85894b